content/src/debug.js

```diff
@@ -1,26 +1,29 @@
-//////////////////////////////////////////////////////////////////////////r
+//////////////////////////////////////////////////////////////////////////
 // DEBUGGER SUPPORT
 ///////////////////////////////////////////////////////////////////////////
 
 var $         = require('jquery'),
     view      = require('view'),
     see       = require('see'),
+    html2canvas = require('html2canvas'),
     sourcemap = require('source-map');
 
-
 eval(see.scope('debug'));
 
-var targetWindow = null;      // window object of the frame being debugged.
-var currentEventIndex = 0;    // current index into traceEvents.
-var currentDebugId = 0;       // id used to pair jquery-turtle events with trace events.
-var debugRecordsByDebugId = {}; // map debug ids -> line execution records.
-var debugRecordsByLineNo = {};  // map line numbers -> line execution records.
-var cachedParseStack = {};    // parsed stack traces for currently-running code.
-var pollTimer = null;         // poll for stop button.
-var stopButtonShown = 0;      // 0 = not shown; 1 = shown; 2 = stopped.
-var currentSourceMap = null;  // v3 source map for currently-running instrumented code.
-var traceEvents = [];         // list of event location objects created by tracing events
-var stuckTime = null;         // timestmp to detect stuck programs
+var targetWindow = null;       // window object of the frame being debugged.
```

Comment:
Let's do the following here.  Instead of having bindFrame reset this stuff and separately it's also initialized here, make a resetDebugState function that is called by both bindFrame and at initialization time.  That way we can make sure that everything is set up the same way both at the start and when restarting debugging.